hogben/visualise.py

```diff
@@ -110,6 +115,9 @@ def angle_choice_with_time(
     # Check that the angle can be varied for the sample.
     assert isinstance(sample, VariableAngle)
 
+    # Set contrasts to empty list if not provided
+    contrasts = contrasts or []
```

Comment:
Is it not better to set the default in the init to contrasts: Optional[list] = [], this evaluates to None if required and removes this line